simulations/vip-diamond-comptroller/vip-diamond-comptroller/utils/cut-params.json

```diff
@@ -0,0 +1,3 @@
+{
+  "cutParams": []
```

Comment:
We need this

Reply:
yes will be adding it. Once the Diamond for mainnet will be deployed.